app/views/pages/download.html.erb

```diff
@@ -6,10 +6,10 @@
     RubyGems is a package management framework for Ruby. Download the latest version here:
   </p>
   <div id="formats" class="download__formats">
-    <%= link_to "tgz", "http://production.cf.rubygems.org/rubygems/rubygems-#{version_number}.tgz", class: "download__format" %>
-    <%= link_to "zip", "http://production.cf.rubygems.org/rubygems/rubygems-#{version_number}.zip", class: "download__format" %>
-    <%= link_to "gem", "http://production.cf.rubygems.org/rubygems/rubygems-update-#{version_number}.gem", class: "download__format" %>
-    <%= link_to "git", "http://github.com/rubygems/rubygems", class: "download__format" %>
+    <%= link_to "tgz", "https://rubygems.global.ssl.fastly.net/rubygems/rubygems-#{version_number}.tgz", class: "download__format" %>
+    <%= link_to "zip", "https://rubygems.global.ssl.fastly.net/rubygems/rubygems-#{version_number}.zip", class: "download__format" %>
+    <%= link_to "gem", "https://rubygems.global.ssl.fastly.net/rubygems/rubygems-update-#{version_number}.gem", class: "download__format" %>
```

Comment:
This is something we have to think about a lot at SoundCloud, where we dynamically switch between CDNs for streaming audio based on factors such as price, availability, and requesting IP address (e.g. in our testing, some CDNs have dramatically better performance to Australia and parts of Asia). Having a canonical URL gives us a lot more flexibility during a CDN outage (or when a particular CDN’s IPs are block by an entire country). In our experience, this flexibility is worth paying the price of making an extra hop.